### PR TITLE
This Month in Xilem,  June 2024

### PR DESCRIPTION
**Pull request by DJMcNab**
Lots of significant updates in this one

### CODE REVIEW

File: content/blog/2024-07-02-tmix-06/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+An in-progress Emoji viewer example ([xilem#420][]) running on Android.
+This example currently only works for systems with COLR system emoji, such as some Android systems.
+
+</figcaption>
+</figure>
+
+[xilem#310][] brought in some new ideas about how Xilem Core can be structured, to move away from the macro heavy design used previously.
+I believe that this has additional advantages for specialised `View` types, such as OS windows, although this is yet to be integrated.
+Support for the `Adapt` views built on this has enabled an [Elm](https://elm-lang.org/)-inspired example for Xilem in [xilem#401][], showing the flexibility of the Xilem architecture.
+Our observability support has also been improved, with automatic tracing to a file in debug builds added in [xilem#384][]; [xilem#396][] will lay the foundation for more advanced debugging capabilities.
```
Comment (PoignardAzur) [inline content/blog/2024-07-02-tmix-06/index.md:25]: xilem#396 isn't really about tracing or debugging. It's more of an internal architecture change, for making easier to write new features and slightly easier testing.

**Comment by PoignardAzur** [comment]
You know, I'm keep thinking I'm late on stuff, but looking at the Masonry part of the three-months roadmap, I actually think we're well on-track to completing it by the end of the month. Huh.

File: content/blog/2024-07-02-tmix-06/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
++++
+title = "This Month in Xilem, June 2024"
+authors = ["Daniel McNab"]
++++
+
+We released [Vello 0.2.0][] at the start of the month, which included some key improvements discussed in previous months' updates.
+In June, we merged initial handling of Emoji ([vello#615][]), which supports glyphs which use the COLR specification (so this does not, for example, support Apple's system Emoji).
+In-flight work includes an enhanced testing setup ([vello#610][]), GPU memory allocation robustness ([vello#606][]), and improved correctness for some unusual strokes ([vello#607][]).
+
```
[the remark this answers is not in the document]
Comment (raphlinus) [reply content/blog/2024-07-02-tmix-06/index.md:9]: Yes, how bout this:

The [GPU-Friendly Stroke Expansion](https://github.com/linebender/gpu-stroke-expansion-paper) paper has been accepted to [High Performance Graphics 2024](https://www.highperformancegraphics.org/2024/index.html), and will be presented in Denver on July 26. A revised draft of the paper is published on [arXiv](https://arxiv.org/abs/2405.00127).

File: content/blog/2024-07-02-tmix-06/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<figcaption>
+
+Screenshot by Veniamin Ilmer of their Minesweeper game, used under the [Apache 2.0](https://github.com/veniamin-ilmer/minesweeper_xilem/blob/main/LICENSE) license.
+
+</figcaption>
+</figure>
+
+Following on from May, Olivier wrote and published a [report](@/blog/2024-06-15-rustnl-2024-unconference.md) on our experience of the RustNL unconference.
+This led to some useful discussion online - see [the zulip thread](https://xi.zulipchat.com/#narrow/stream/181284-blogging/topic/Draft.20-.20Report.20on.20the.20RustNL.202024.20Conference/near/444974910) corresponding with that post for links.
+We are now in the final month of the [May-July 2024 roadmap](@/blog/2024-06-15-rustnl-2024-unconference.md).
```
Comment (jaredoconnell) [inline content/blog/2024-07-02-tmix-06/index.md:49]: There doesn't seem to be much roadmap discussion on that page. Did you mean [2024-06-17-roadmap-may-2024.md](https://github.com/linebender/linebender.github.io/blob/main/content/blog/2024-06-17-roadmap-may-2024.md)?

File: content/blog/2024-07-02-tmix-06/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
++++
+title = "This Month in Xilem, June 2024"
+authors = ["Daniel McNab"]
++++
+
+We released [Vello 0.2.0][] at the start of the month, which included some key improvements discussed in previous months' updates.
+In June, we merged initial handling of Emoji ([vello#615][]), which supports glyphs which use the COLR specification (Apple's system emoji font does not use this format and so currently will not render).
```
Comment (jaredoconnell) [inline content/blog/2024-07-02-tmix-06/index.md:7]: Mentioning "In June" is odd in the context of a June update. It makes you wonder if this isn't a June update.

**Review by jaredoconnell — COMMENTED** [review]
Looks good. I just have a few comments.

**Review by jaredoconnell — APPROVED** [review]

**Pull request merged**